app/Jobs/Migrations/StartIntegrationMigration.php

```
@@ -123,6 +128,72 @@ protected function startGitHub(): void
+        // Transactions: page backwards using since cursor by 90-day windows until empty
+        $now = Carbon::now();
+        $contextTx = [
+            'service' => 'monzo',
+            'instance_type' => 'transactions',
+            'cursor' => [
+                'end_iso' => $now->toIso8601String(),
+                'window_days' => 89,
+            ],
+            'timebox_until' => $this->timeboxUntil?->toIso8601String(),
+        ];
+        $contextPots = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$contextPots'. (Unused Code Rules)

(UnusedLocalVariable)